app/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019-2024, Arm Limited and Contributors
+# Copyright (c) 2019-2025, Arm Limited and Contributors
```

Comment:
Changed nothing but the copyright here?

Reply:
Yeah, that was a desparate try to fix CI. Will revert.